azureml_deployment/src/test.py

```diff
@@ -0,0 +1,19 @@
+import requests
+
+# Mock data
+body = {
+    "data": [
+        {"x1": 1.0, "x2": -1.0},
+        {"x1": -1.0, "x2": 1.0},
+    ]
+}
+
+# Create a post request to check the web server.
+local_server_address = "http://localhost:8000/score"
```

Comment:
This only works locally.
You also want to run this against your really deployment.
Maybe make the host configurable via the argparser?